README.md

```diff
@@ -1,22 +1,88 @@
-# labs-authorization
-[![Travis][BadgeTravis]][Travis]
+# Udaru
+[![npm][npm-badge]][npm-url]
+[![travis][travis-badge]][travis-url]
+[![coveralls][coveralls-badge]][coveralls-url]
+[![david][david-badge]][david-url]
 
+Udaru is a policy based authorization module that can be used to add permissions to 'actions' and 
+'resources'. Udaru supports 'organisations', 'teams', and 'users'; policies can be created for 
+each. Udaru can be used as a [stand-alone module]() , or as a [stand-alone server]() or 
+[Hapi plugin](). This repository contains the code for all three running configurations.
 
-A component providing authorization functionality
+## Install
+To install via npm,
+
+```
+npm install udaru
+```
+
+## Testing, benching & linting
+Before running tests, ensure a valid Postgres database is running. The simplest way to do this is
+via Docker. Assuming docker is intstalled on your machine, in the root folder, run,
+
+```
+docker-compose up
```

Comment:
@p16 I'd honestly prefer to drop that command. People are more likely to understand what is happening better by giving them the correct command rather than hiding it.

Thoughts?